packages/py3_pip.rb

```diff
@@ -0,0 +1,32 @@
+require 'package'
+
+class Py3_pip < Package
```

Comment:
Won't this conflict with the pip installed in python3 and py2_pip?  I don't think we need either py#_pip package, actually.

Reply:
the pip installed with python3 I deleted.

Reply:
I don't think we should do that.  We should install one or the other.  If we want to keep these packages, we should make them dependencies of python27 and python3 and remove the pip install there.

Reply:
I don't think all users need pip. It doesn't need to be in the python package at all, it can just be another pacakge we provide for people that can't find the pypi package in they need in our repositories. Plus it's bloating the core packages.

Reply:
Respectfully disagree on removing pip/pip3. I use it all the time. I know that it is nice to have compiled python packages, but I think that we need to keep pip, because people are going to install it anyways using `get-pip.py`. 

Also since it is part of the default python install, I think people will expect it to be there, and we will have to deal with support issues from people complaining about that.

Reply:
@satmandu: Do you think we should keep the python3 package intact and not create the py2_pip and py3_pip packages?

Reply:
@uberhacker Yes, I think that python3 should include the pip3 install as part of the postinstall. But that doesn't mean that you can't have a py3_pip package too...

Reply:
yes, so we should just not install pip as a dependency of python. With the way we bootstrap pip, chromebrew can't track it through a destdir and can't remove it, even after the user runs `crew remove python3`. The pip package, on the other hand, has trackable files. It's the obvious better choice.